plotly/tools.py

```diff
@@ -364,17 +369,10 @@ def embed(file_owner_or_url, file_id=None, width="100%", height=525):
         pass
     if _ipython_imported:
         if file_id:
-            plotly_domain = (
-                session.get_session_config().get('plotly_domain') or
-                get_config_file()['plotly_domain']
-            )
-            url = "{plotly_domain}/~{un}/{fid}".format(
-                plotly_domain=plotly_domain,
-                un=file_owner_or_url,
-                fid=file_id)
+            return PlotlyDisplay(file_owner_or_url, file_id, share_key, width,
+                                 height)
         else:
-            url = file_owner_or_url
-        return PlotlyDisplay(url, width, height)
+            return PlotlyDisplay(file_owner_or_url, width, height)
```

Comment:
the get_embed can take a first parameter of `file_owner_or_url`, this is for more flexibility / backwards compat at one point. however, PlotlyDisplay doesn't *do* anything with file_id, this is because it *expects* that the first positional argument is a url, not a username of some kind.

I think we shouldn't add `file_id` to the `PlotlyDisplay`, is there a reason for it?